config/transport_tasks_config.rb

```diff
@@ -40,8 +41,8 @@
 threads 0, config['concurrency']
 
 impl = ACE::TransportApp.new(config)
-# unless config.whitelist.nil?
-#   impl = ACE::ACL.new(impl, config.whitelist)
-# end
+unless config['whitelist'].nil?
+  impl = ACE::ACL.new(impl, config['whitelist'])
```

Comment:
can we just use `Bolt::ACL`?

Reply:
`Statement/Question: We could alternatively reuse the BoltServer::ACL class?`

Happy to reuse